.gitignore

```diff
@@ -31,3 +31,4 @@ msvc/Release/
 .*.swp
 tags
 mkmf.log
+.vs
```

Comment:
Not a project maintainer, but this addition to the `.gitignore` should definitely not be part of the same commit as the changes in `src/merge_driver.c`.